docs/rule_severity.rst

```diff
@@ -0,0 +1,68 @@
+Rule Severity
+=============
+
+VSG supports rule severity with two built in levels:  Error and Warning.
+The default behavior for most rules is **Error**.
+Only the **Error** severity level will result in an exit status of 1.
+**Errors** will also be the only errors written to a JUnit XML file if that option is chosen.
+
+The table below summarizes the built in severities:
+
++-------------+---------+-------------+-------+-----------+---------------------------------------------------------------+
+| Name        |  Type   | Exit Status | JUnit | Syntastic | Description                                                   |
++=============+=========+=============+=======+===========+===============================================================+
+| Error       |  error  |      1      |  Yes  |   ERROR   | A rule which must be fixed.                                   |
++-------------+---------+-------------+-------+-----------+---------------------------------------------------------------+
+| Warning     | warning |      0      |  No   |  WARNING  | A rule which should be fixed if it is reasonable to do so.    |
++-------------+---------+-------------+-------+-----------+---------------------------------------------------------------+
+
+Configuring Severity Levels
+---------------------------
+
+The existing severity level of a rule can be configured.
+For example, if you want to change the line length rule, *length_001*, to a **Warning** instead of an **Error**, use the following configuration:
+
+.. code-block:: json
+
+   {
+       "rule":{
+           "length_001":{
+               "severity": "Warning"
+           }
+       }
+   }
+
+Defining Severity Levels
+------------------------
+
+VSG supports defini
```

Comment:
Looks like you forgot to finish the comments here.

Reply:
agree, not sure what happened there.  Probably distracted by something shiny.